pyproject.toml

```diff
@@ -139,15 +139,17 @@ source_pkgs = ["shap"]
 combine = ["shap", "*/site-packages/shap"]
 
 [tool.cibuildwheel]
-skip = ["pp*", "*-musllinux_*"]
+# Restrict the set of builds to mirror the wheels available in scikit-learn. See #3028
+skip = ["pp*", "*-musllinux_aarch64", "cp38-musllinux_x86_64"]
```

Comment:
What is the motivation behind the two new specific skips here?

Also, the comment about mirroring scikit-learn is probably no longer relevant, as we're now choosing to build a wider range of wheels.

Reply:
I added comments that describe skips here. `*-musllinux_aarch64` is not provided since scipy/numpy doesn't provide them. We would need to build numpy in the wheel-building process to compile SHAP. `cp38-musllinux_x86_64` is not provided since numpy never had cp38 (musllinux) wheels. They introduced musllinux wheels after dropping cp38. The later can be removed when SHAP drop Python 3.8, which can be safely done in my opinion and according to NEP https://numpy.org/neps/nep-0029-deprecation_policy.html.

Reply:
> Also, the comment about mirroring scikit-learn is probably no longer relevant, as we're now choosing to build a wider range of wheels.

It is true. I changed the comment from scikit to scipy. As I see in #3028 it was already discussed that we provide wheels that scipy has (Numpy has everything that scipy has +win32 which was not included as discussed in #3028).